eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -34,6 +34,7 @@ steps:
       path: $(VCPKG_INSTALLATION_ROOT)
       cacheHitVar: VcpkgRestoredFromCache
     displayName: Vcpkg Cache
+    timeoutInMinutes: 10
```

Comment:
Is there an reason to explicitly set this to 10? I believe the default is 60.

Reply:
The rationale should be placed in a comment in this file.